packages/specs/package.json

```diff
@@ -36,8 +36,6 @@
 	"homepage": "https://github.com/directus/directus#readme",
 	"files": [
 		"dist",
-		"LICENSE",
-		"README.md",
```

Comment:
Isn't the readme required for rendering on npm, or does it send the readme no matter what?

Reply:
Yes, the readme ([as well as other files](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#files)) is always included 👍